_data/retail.yml

```
@@ -397,18 +478,25 @@ websites:
     twitter: Staples
     facebook: staples
     img: staples.png
+    regions:
+      - us
+      - ca
 
   - name: Target
     url: https://www.target.com/
     twitter: Target
     img: target.png
+    regions:
+      - us
 
   - name: Thalia
     url: https://www.thalia.de/
     facebook: thalia.de
     email_address: info@thalia.de
     img: thalia.png
     lang: de
+    regions:
+      - de
```

```suggestion
    lang: de
```
Thalia ships to [17 countries](https://www.thalia.de/shop/hilfe-versand/show/#D2).